server/plugin.go

```diff
@@ -506,6 +510,9 @@ func (p *Plugin) OnActivate() error {
 		)
 		p.store = timerlayer.New(store, p.GetMetrics())
 		if err = p.store.Init(); err != nil {
+			if err2 := p.store.Shutdown(); err2 != nil {
```

Comment:
Not blocking but a comment...  with so many exit paths and the chance to forget to shut down the DB in the future, it would be nice to shutdown the DB anytime an err is returned from this method.  Either put all this logic in a setup method called by `OnActivate`, or use a defer and err return var.

Reply:
:+1: